app/controllers/host_shelters/properties_controller.rb

```diff
@@ -15,13 +15,43 @@ def create
     end
   end
 
+  def index
+    @properties = current_user.properties
+  end
+
   def show
+    property_owner(property_id_params)
+    @property = Property.find(property_id_params)
+  end
+
+  def status_accepted
+    reservation = Reservation.find(params[:property_id])    
```

Comment:
@erose357 why are we finding a Reservation by `:property_id` and not `:reservation_id?`

Reply:
Spoke to Eddie in person. Apparently `:property_id` is actually the correct id for `reservations` but is coming through as `:property_id` due to namespacing